hr_employee_lastnames/__manifest__.py

```diff
@@ -8,7 +8,7 @@
     "license": "AGPL-3",
     "category": "Human Resources",
     "summary": "Split Name in First Name, Father's Last Name and Mother's Last Name",
-    "depends": ["hr_employee_firstname"],
+    "depends": ["hr_employee_firstname", "partner_second_lastname"],
```

Comment:
Note. This dependency was present in the beginning.

https://github.com/OCA/hr/pull/880/files#diff-7181f4938268a9cb4df7d9dbaa8e9b4a764272212e4563146827e999b581ba8dR14